### PR TITLE
Add user endpoint

### PR DESCRIPTION
**Pull request by dzhou343**
Endpoint route works and is tested via Postman. 

I am getting the current error dump when trying to Post the sample input from the pipeline specifications page from [Notion](https://www.notion.so/holmusk/Pipeline-specifications-e8f3648483404c04aaf04a69bfdccb1c). 

Error Dump: 

```
(node:6701) UnhandledPromiseRejectionWarning: Neo4jError: Invalid input '8': expected whitespace, a variable, node labels, a property map, ')' or PatternElement (line 2, column 9 (offset: 47))
"CREATE (80008000:handphone:contact:number {value: '80008000'})"
         ^

    at captureStacktrace (/home/daniel/HolmuskDB/backend/node_modules/neo4j-driver/lib/result.js:263:15)
    at new Result (/home/daniel/HolmuskDB/backend/node_modules/neo4j-driver/lib/result.js:68:19)
    at newCompletedResult (/home/daniel/HolmuskDB/backend/node_modules/neo4j-driver/lib/transaction.js:449:10)
    at Object.run (/home/daniel/HolmuskDB/backend/node_modules/neo4j-driver/lib/transaction.js:287:14)
    at Transaction.run (/home/daniel/HolmuskDB/backend/node_modules/neo4j-driver/lib/transaction.js:123:32)
    at /home/daniel/HolmuskDB/backend/src/db-utils.js:21:54
    at TransactionExecutor._safeExecuteTransactionWork (/home/daniel/HolmuskDB/backend/node_modules/neo4j-driver/lib/internal/transaction-executor.js:134:22)
    at TransactionExecutor._executeTransactionInsidePromise (/home/daniel/HolmuskDB/backend/node_modules/neo4j-driver/lib/internal/transaction-executor.js:122:32)
    at /home/daniel/HolmuskDB/backend/node_modules/neo4j-driver/lib/internal/transaction-executor.js:61:15
    at new Promise (<anonymous>)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6701) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:6701) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

This occurs on the following JSON request body input: 
```
{ "type" : "create"
, "name" : "Jonas"
, "details" : [ { "relations" : []
                , "term" : 80008000
                , "labels" : ["handphone", "contact", "number"]
                }
              , { "relations" : ["From"]
                , "term" : "ABC Group"
                , "labels" : ["Company"]
                }
              ]
}
```

EDIT: Tracked in #36

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+tailwind.css
```
Comment (kwannoel) [inline client/.gitignore:7]: This is in the latest master, I think you need to rebase against it. In your current branch just do:
```
git pull --rebase origin master
```

Comment (dzhou343) [reply client/.gitignore]: rebased and force pushed latest commit. 

Raised issue #36 pertaining to the error encountered above regarding querying.

File: backend/src/server.js [hunk 2]
```diff
@@ -1,16 +1,26 @@
 const express = require('express')
+const dbq = require('./db-queries')
```
Comment (kwannoel) [inline backend/src/server.js:2]: Use clearer variable name here. Can just be `dbQueries`

File: backend/src/server.js [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
 
 function runServer (driver) {
   const app = express()
   const port = process.env.PORT || 5000
 
   // console.log that your server is up and running
   app.listen(port, () => console.log(`Listening on port ${port}`))
+  
+  app.use(express.json())
```
Comment (kwannoel) [inline backend/src/server.js:11]: What's the functionality of this middleware?

File: backend/src/server.js [hunk 4]
```diff
@@ -1,16 +1,26 @@
...
   })
+
+  app.post('/db/add/person', (req, res) => { 
+    dbq.addPerson(driver,req.body).then(res.send(req.body)) 
+  })
+    
 }
 
 module.exports = { runServer }
+
```
Comment (kwannoel) [inline backend/src/server.js:25]: strip newlines

Comment (kwannoel) [reply backend/src/server.js:25]: Run `npm run standard` before pushing branch

File: backend/src/server.js (hunk 3 above)
Comment (dzhou343) [reply backend/src/server.js:11]: Allows req.body to be processed in JSON. Will add comment to clarify and amend commit.

**Pull request merged**